src/Domain.LinnApps/Parts/PartService.cs

```diff
@@ -345,17 +345,35 @@ public Part CreateFromSource(int sourceId, int createdBy, IEnumerable<PartDataSh
                                       { "address", bcc2.EmailAddress }
                                   }
                           };
-            this.emailService.SendEmail(
-                to.EmailAddress,
-                to.User.Name,
-                cc,
-                null,
-                "stores@linn.co.uk",
-                "Parts Utility",
-                $"New Source Sheet Created - {source.PartNumber}",
-                $"Click here to view: https://app.linn.co.uk/parts/sources/{source.Id}",
-                null,
-                null);
+            if (source.MechanicalOrElectrical == "E")
+            {
+                this.emailService.SendEmail(
+                    to.EmailAddress,
+                    to.User.Name,
+                    cc,
+                    null,
+                    "stores@linn.co.uk",
+                    "Parts Utility",
+                    $"New Source Sheet Created - {source.PartNumber}",
+                    $"Click here to view: https://app.linn.co.uk/parts/sources/{source.Id}",
+                    null,
+                    null);
+            }
+            else
```

Comment:
i would do else if source.MechnicalOrElectrical == "M" here just to be specific. I guess in theory its still possible to create a source sheet that is neither Electrical or Mechanical... although can't really think why that would ever happen